src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int NUMBER = 100;
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball(new Random().nextInt(NUMBER), randomColor.getRandomColor());
+        return ball;
```

Comment:
```suggestion
        return new Ball(new Random().nextInt(NUMBER), randomColor.getRandomColor());
```
In situations like this, there is no need to store the object in a local variable if it will be immediately used. Instead, you can pass the expression directly without assigning it to a variable